backend/app/main.py

```
@@ -1,9 +1,25 @@
 from fastapi import FastAPI
+from app.router.instance import api_router
 
 app = FastAPI()
 
 @app.get("/")
 def read_root():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)